dom-events/buttons.js

```diff
@@ -0,0 +1,26 @@
+// click event
+function handleClick(event) {
+  console.log('button clicked');
+  console.log(event);
+  console.log(clickButton);
```

Comment:
Instead of logging out the original variable holding the element, you should log out `event.target` in these functions